spec/models/organization_spec.rb

```diff
@@ -71,7 +71,9 @@
 
     it "downcases slug" do
       organization.slug = "HaHaHa"
-      organization.save
+      perform_enqueued_jobs do
```

Comment:
Does it actually need to perform jobs to check that the slug was downcased?

Reply:
No, I mixed 2 things because I wanted to have the cache busting triggered by the save method. It was not good, I left the downcasing alone and added another test.